frontend/src/pages/userDashboard.tsx

```
@@ -38,48 +38,58 @@ const UserDashboard: React.FC = () => {
+            <FiSearch className="w-5 h-5 text-gray-400" />
+          </div>
+          
+          {showSearchResults && (
+            <div 
+              className="absolute top-12 left-0 w-full bg-white shadow-lg rounded-lg p-2 z-50"
+              onMouseLeave={() => setShowSearchResults(false)}
+            >
+              {isLoading ? (
+                <div className="p-4"><LoadingSpinner /></div>
+              ) : (
+                <ul role="listbox" className="max-h-48 overflow-y-auto">
```

Biome (1.9.4)

[error] The HTML element with the interactive role "listbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "listbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)